plugins/modules/apk.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# Copyright (c) 2024 Krzysztof Bialek/Markus Weippert
+# GNU General Public License v3.0 (see https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+DOCUMENTATION = r"""
+module: apk
+short_description: Manage packages with apk on OpenWrt
+description:
+  - The M(community.openwrt.apk) module manages packages on OpenWrt using the apk package manager (available in OpenWrt 24.10+).
```

Comment:
I believe this should be:
```suggestion
  - The M(community.openwrt.apk) module manages packages on OpenWrt using the apk package manager (available in OpenWrt 25.12+).
```